apps/metric_vnode/eqc/metric_vnode_eqc.erl

```diff
@@ -181,23 +178,23 @@ prop_handoff() ->
                    os:cmd("mkdir data"),
                    {S, T} = eval(D),
 
-                   List = ?T:to_list(T),
+                   List = gb_trees:to_list(T),
 
                    List1 = [{get(S, Time, 1), V} || {Time, V} <- List],
                    List2 = [{unlist(mmath_bin:to_list(Vs)), Vt} || {{_ ,Vs}, Vt} <- List1],
```

Comment:
Line 184 is too long:                    List2 = [{unlist(mmath_bin:to_list(Vs)), Vt} || {{_ ,Vs}, Vt} <- List1],.